src/Avalonia.Base/Input/PointerPoint.cs

```diff
@@ -155,17 +160,18 @@ public PointerPointProperties(RawInputModifiers modifiers, PointerUpdateKind kin
         }
 
         public PointerPointProperties(RawInputModifiers modifiers, PointerUpdateKind kind,
-            float twist, float pressure, float xTilt, float yTilt
+            float twist, float pressure, float xTilt, float yTilt, Rect contactRect
```

Comment:
Why do we need to extend the API? Can't relevant XI2 properties be converted to existing ones? How other platforms expose the same information? How browsers are handling that?

Reply:
How the same info is exposed by UWP?

Reply:
@kekekeks It can be implemented in Windows platform, see https://learn.microsoft.com/en-us/windows-hardware/design/component-guidelines/touchscreen-required-hid-top-level-collections

Reply:
Also, we can get the touch size in wpf by [`StylusPointProperties.Width`](https://learn.microsoft.com/en-us/dotnet/api/system.windows.input.styluspointproperties.width) and [`StylusPointProperties.Height`](https://learn.microsoft.com/en-us/dotnet/api/system.windows.input.styluspointproperties.height)

Reply:
> How other platforms expose the same information? How browsers are handling that?

I support the windows platform now.